keystore/keystore.go

```diff
@@ -52,6 +57,24 @@ func validateName(name string) error {
 	return nil
 }
 
+// NewKeystore returns a Keystore using the default implementation.
+func NewKeystore(dir string) (Keystore, error) {
+	return NewEncodedFSKeystore(dir)
+}
+
+// NewEncodedFSKeystore is returns a filesystem-backed keystore which encodes
+// key names using base32.
+func NewEncodedFSKeystore(dir string) (*EncodedFSKeystore, error) {
```

Comment:
I'd just have one keystore. Supporting both seems like extra and unnecessary work.

Reply:
this was an old PR that was already approved so I just did minimal maintenance, but ok.